test/e2e/serial/tests/scheduler_cache_stall.go

```diff
@@ -0,0 +1,371 @@
+				for _, testPod := range testPods {
+					err := fxt.Client.Create(context.TODO(), testPod)
+					Expect(err).ToNot(HaveOccurred())
+				}
+				// note the cleanup is done automatically once the ns on which we run is deleted - the fixture takes care
+
+				// very generous timeout here. It's hard and racy to check we had 2 pods pending (expected phased scheduling),
+				// but that would be the most correct and stricter testing.
+				failedPods, updatedPods := wait.With(fxt.Client).Timeout(3*time.Minute).ForPodListAllRunning(context.TODO(), testPods)
+				if len(failedPods) > 0 {
+					nrtListFailed, _ := e2enrt.GetUpdated(fxt.Client, nrtv1alpha2.NodeResourceTopologyList{}, time.Minute)
+					klog.Infof("%s", intnrt.ListToString(nrtListFailed.Items, "post failure"))
+
+					for _, failedPod := range failedPods {
+						_ = objects.LogEventsForPod(fxt.K8sClient, failedPod.Namespace, failedPod.Name)
+					}
+				}
+				Expect(len(failedPods)).To(BeZero(), "unexpected failed pods: %q", accumulatePodNamespacedNames(failedPods))
+
+				for _, updatedPod := range updatedPods {
+					schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+				}
+			})
+
+			DescribeTable("[nodeAll] against all the available worker nodes",
+				// like non-regression tests, but with jobs present
+
+				func(setupPod setupPodFunc) {
+					timeout := nroSchedObj.Spec.CacheResyncPeriod.Round(time.Second) * 10
+					klog.Infof("pod running timeout: %v", timeout)
+
+					nrts := e2enrt.FilterZoneCountEqual(nrtList.Items, 2)
+					if len(nrts) < 1 {
+						e2efixture.Skip(fxt, "Not enough nodes found with at least 2 NUMA zones")
+					}
+
+					// CAUTION here: we assume all worker node identicals, so to estimate the available
+					// resources we pick one at random and we use it as reference
+					nodesNames := e2enrt.AccumulateNames(nrts)
+					referenceNodeName, ok := e2efixture.PopNodeName(nodesNames)
+					Expect(ok).To(BeTrue())
+
+					klog.Infof("selected reference node name: %q", referenceNodeName)
+
+					nrtInfo, err := e2enrt.FindFromList(nrts, referenceNodeName)
+					Expect(err).ToNot(HaveOccurred())
+
+					// we still are in the serial suite, so we assume;
+					// - even number of CPUs per NUMA zone
+					// - unloaded node - so available == allocatable
+					// - identical NUMA zones
+					// - at most 1/4 of the node resources took by baseload (!!!)
+					// we use cpus as unit because it's the easiest thing to consider
+					resQty := e2enrt.GetMaxAllocatableResourceNumaLevel(*nrtInfo, corev1.ResourceCPU)
+					resVal, ok := resQty.AsInt64()
+					Expect(ok).To(BeTrue(), "cannot convert allocatable CPU resource as int")
+
+					cpusVal := (10 * resVal) / 8
```

Comment:
can you elaborate on why this is calculated like this please?

Reply:
it's "a little more" than the max allocatable, to trigger overallocation and force the scheduler to wait for un update